src/test/java/org/scijava/script/ScriptFinderTest.java

```diff
@@ -241,6 +243,11 @@ public ScriptEngine getScriptEngine() {
 			throw new IllegalStateException();
 		}
 
+        @Override
+        public AutoCompleter getAutoCompleter(ScriptEngine scriptEngine) {
+            return new DefaultAutoCompleter(this, scriptEngine);
```

Comment:
This override is unnecessary, no? Because `AbstractScriptLanguage` already defines this same logic?